src/theme/DocItem/index.js

```diff
@@ -73,6 +88,14 @@ function DocItem(props) {
                   </span>
                 </div>
               )}
+
+              {renderTocMobile && (
+                <TOCCollapsible
```

Comment:
changes needed to render this

![image](https://user-images.githubusercontent.com/749374/125669003-3bfcf1ec-c51d-4935-a58b-9413dafe4950.png)